src/test/java/proj/kedabra/billsnap/presentation/controllers/BillControllerIT.java

```diff
@@ -843,7 +843,7 @@ void shouldReturnExceptionIfListEmailsNotEmailFormatInInviteRegisteredResourceGi
     }
```

Comment:
BillControllerIT: missing test cases for null tax name or tax percentage, or is negative number or not in range.